compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lower_to_aie.mlir

```diff
@@ -420,7 +460,7 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
       // expected-error @+1 {{could not convert to AIEDialect ops}}
       amdaie.controlcode {
         // expected-error @+1 {{implicit source/target L3 memref has rank greater than the expected static offsets/sizes/strides rank (4)}}
-        %npu_dma_1 = amdaie.npu.dma_cpy_nd %dma_target_l3([] [] [], [] [] [])
+        %npu_dma_1 = amdaie.npu.dma_cpy_nd %dma_target_l3([] [] [] %bd_id_0, [] [] [])
```

Comment:
i don't think it's a good idea that these aren't delimited in some special way _and_ optional - you're gonna have parser ambiguities if/when you try to introduce another operand (which already happens for attrs in [mlir-aie](https://github.com/Xilinx/mlir-aie/pull/1137)). i guess you don't want to require them always because you don't always need to run the subsumption pass but then you _also_ don't want to special/weird syntax.

it's too bad you can't stick value into the attr dict.

Reply:
Good point. Let me try making them required, for example by having an unassigned value.

Reply:
I think I like it more to have them optional so null means unassigned. Instead, I added a `bd_id =` delimiter.

Reply:
Ya named kw is what I thought of too